packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -78,7 +78,42 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
   activeValidationModule!: BaseValidationModule;
 
   private constructor(readonly biconomySmartAccountConfig: BiconomySmartAccountV2Config) {
-    super(biconomySmartAccountConfig);
+    super({
+      ...biconomySmartAccountConfig,
+      chain: getChain(biconomySmartAccountConfig.chainId),
+      rpcClient: biconomySmartAccountConfig.rpcUrl || (RPC_PROVIDER_URLS[biconomySmartAccountConfig.chainId] as string),
+      entryPointAddress: biconomySmartAccountConfig.entryPointAddress as Hex,
+      accountAddress: (biconomySmartAccountConfig.accountAddress as Hex) ?? undefined,
+      factoryAddress: biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS,
+    });
+    this.index = biconomySmartAccountConfig.index ?? 0;
+    this.chainId = biconomySmartAccountConfig.chainId;
+    this.bundler = biconomySmartAccountConfig.bundler;
+    this.implementationAddress = biconomySmartAccountConfig.implementationAddress ?? (BICONOMY_IMPLEMENTATION_ADDRESSES_BY_VERSION.V2_0_0 as Hex);
+
+    if (biconomySmartAccountConfig.biconomyPaymasterApiKey) {
+      this.paymaster = new BiconomyPaymaster({
+        paymasterUrl: `https://paymaster.biconomy.io/api/v1/${biconomySmartAccountConfig.chainId}/${biconomySmartAccountConfig.biconomyPaymasterApiKey}`,
+      });
+    } else {
+      this.paymaster = biconomySmartAccountConfig.paymaster;
+    }
+
+    const defaultFallbackHandlerAddress =
+      this.factoryAddress === DEFAULT_BICONOMY_FACTORY_ADDRESS ? DEFAULT_FALLBACK_HANDLER_ADDRESS : biconomySmartAccountConfig.defaultFallbackHandler;
+    if (!defaultFallbackHandlerAddress) {
+      throw new Error("Default Fallback Handler address is not provided");
+    }
+    this.defaultFallbackHandlerAddress = defaultFallbackHandlerAddress;
+
+    this.provider = createPublicClient({
+      chain: getChain(biconomySmartAccountConfig.chainId),
+      transport: http(biconomySmartAccountConfig.rpcUrl || (RPC_PROVIDER_URLS[biconomySmartAccountConfig.chainId] as string)),
+    });
+
+    this.scanForUpgradedAccountsFromV1 = biconomySmartAccountConfig.scanForUpgradedAccountsFromV1 ?? false;
+    this.maxIndexForScan = biconomySmartAccountConfig.maxIndexForScan ?? 10;
+    // REVIEW: removed the node client
```

Comment:
Why was node client removed ?

Reply:
It was required for v1 client where we used to call a backend service. We don't need any backend service as of now so removing it.